docs/en/extdev-transfer-gateway.md

```diff
@@ -6,12 +6,12 @@ sidebar_label: Transfer Gateway Testnet
 
 ## Overview
 
-In this doc, we'll walk you through the setup required to transfer tokens between token contracts you've deployed to `extdev` and `Rinkeby`. If you haven't done so already you should first read through the high-level overview of the [Transfer Gateway][].
+In this doc, we'll walk you through the setup required to transfer tokens between token contracts you've deployed to Extdev Testnet and `Rinkeby`. If you haven't done so already you should first read through the high-level overview of the [Transfer Gateway][].
```

Comment:
Why are references to Rinkeby formatted differently to Extdev? Rinkeby is a testnet just like Extdev, so the formatting should be consistent and so should the terminology, it's either Rinkeby Tesnet and Extdev Testnet, or just Rinkeby and Extdev, I prefer the latter everywhere but the introductory sentence.

Reply:
Guess they were like this from time immemorial. Nice catch. it's fixed now.